src/serializers/encoder/v15/encode_graph_entities.c

```
@@ -60,6 +60,29 @@ static void _RdbSaveSIVector
+	SerializerIO rdb,
+	SIValue v
+) {
+	// saves map as
+	// unsigned : map key count
+	// key:value
+	// .
+	// .
+	// .
+	// key:value
+
+	uint n = Map_KeyCount(v);
```

cppcheck

[error] Memory leak

(memleak)